odxtools/structure.py

```diff
@@ -1,18 +1,9 @@
 # SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+
 from .basicstructure import BasicStructure
 
 
+@dataclass
 class Structure(BasicStructure):
```

Comment:
This means that `BasicStructure` and `Structure` could be merged into one class

Reply:
yes. I was just too afraid of the ripple effects...

Reply:
I got curious and looked this up in the XSD: there, `STRUCTURE` is identical to `BASIC-STRUCTURE` with an additional optional `IS-VISIBLE` attribute. Seems quite useless to me...